extensions/Cake.Issues.DocFx.yml

```diff
@@ -2,18 +2,22 @@ Type: Addin
 Name: Cake.Issues.DocFx
 NuGet: Cake.Issues.DocFx
 Assemblies:
-- "/**/Cake.Issues.DocFx.dll"
+- /**/Cake.Issues.DocFx.dll
 Repository: https://github.com/cake-contrib/Cake.Issues.DocFx.git
 ProjectUrl: https://cakeissues.net/
 Author: BBT Software AG
-Description: |-
+Description: >-
   The DocFx support for the Cake.Issues addin allows you to read warnings from DocFx log files.
 
+
   This addin provides the aliases for reading warnings from DocFx log files and provides them to the Cake.Issues addin.
+
```

Comment:
I don't think these extra lines matter when rendering the HTML, but would be nice if they were written exactly as it is in the nuspec https://github.com/cake-contrib/Cake.Issues.DocFx/blob/develop/nuspec/nuget/Cake.Issues.DocFx.nuspec#L10-L19